automation_tools/__init__.py

```diff
@@ -1725,6 +1725,8 @@ def katello_installer(debug=False, sam=False, verbose=True, **kwargs):
             extra_options.append(
                 '--capsule-dns-forwarders="{0}"'.format(forwarder))
 
+    debug = (debug or os.environ.get('DEBUG', '') == 'true')
```

Comment:
Maybe product_install is the better place to put this. I mean, add the logic to get the environment variable and pass the value when It calls katello_installer.

This is a helper function and there is not environment variable processing in here. Because that I feel that we should leave it as a helper function and avoid the introduction of environement variables leaving just parameter way to handle options.